examples/window.rs

```diff
@@ -10,9 +13,10 @@ fn main() {
         .with_title("A fantastic window!")
         .build(&event_loop)
         .unwrap();
+    console!(log, "Built window!");
 
     event_loop.run(|event, _, control_flow| {
-        println!("{:?}", event);
+        console!(log, format!("{:?}", event));
```

Comment:
You might want to revert the changes in this file.

Reply:
Yup, before merge I'll be doing some cleanup (this is still actively in development and I use this example to test.)